### PR TITLE
Remove CompilerUtil

### PR DESCRIPTION
**Pull request by jdahm**
## Technical Description

Removes the CompilerUtil testing class.

### Dependencies

Builds on #886.

### CODE REVIEW

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
@eddie-c-davis @twicki This PR should be ready to be reviewed and merged. It interferes with many of the other test-related PRs, so it would be good to set high priority.

File: dawn/test/unit-test/dawn/CodeGen/CMakeLists.txt [hunk 1]
```diff
@@ -11,8 +11,16 @@
 ##  See LICENSE.txt for details.
 ##
 ##===------------------------------------------------------------------------------------------===##
+include(GoogleTest)
 
 file(COPY input DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 file(COPY reference DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-add_subdirectory(Cuda)
-add_subdirectory(Naive)
+
```
Comment (twicki) [inline dawn/test/unit-test/dawn/CodeGen/CMakeLists.txt:18]: the folder structure should have remained that way (see mirroring the src dir)

File: dawn/test/integration-test/dawn4py-tests/data/hori_diff_stencil_ref.cpp [hunk 2]
```diff
@@ -92,7 +92,7 @@ if(threadIdx.y < +6) {
   int idx111 = (blockIdx.x*32+iblock)*1+(blockIdx.y*4+jblock)*stride_111_1;
   int ijcacheindex= iblock + 1 + (jblock + 1)*34;
 
-  // jump iterators to match the intersection of beginning of next interval and the parallel execution block 
+  // jump iterators to match the intersection of beginning of next interval and the parallel execution block
```
[the remark this answers is not in the document]
Comment (jdahm) [reply dawn/test/integration-test/dawn4py-tests/data/hori_diff_stencil_ref.cpp:95]: Yep. Changed code generation too. You're right though, tests fail if these don't match.

[the remark this answers is not in the document]
Comment (twicki) [reply dawn/test/integration-test/dawn4py-tests/data/hori_diff_stencil_ref.cpp:95]: :+1:

File: dawn/test/unit-test/dawn/CodeGen/CMakeLists.txt (hunk 1 above)
Comment (jdahm) [reply dawn/test/unit-test/dawn/CodeGen/CMakeLists.txt:18]: Good catch!

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
launch jenkins

**Pull request merged**